trunk/SUAVE/Input_Output/OpenVSP/vsp_write.py

```diff
@@ -85,7 +85,7 @@ def write(vehicle,tag):
 
     Properties Used:
     N/A
-    """    
+    """     
```

Comment:
In general try not to change files accidentally. It keeps the commit history clean.